src/html/popup.html

```diff
@@ -237,59 +223,49 @@ <h5 id="addTemplateTitle">Add Default or Custom Template</h5>
+                    </div>
+                    <div class="col s2 right-align">
+                            <a id="clearCustomDomains" class="btn-floating btn-jti tooltipped" data-position="left" data-delay="50" data-tooltip="Clear All Domains">
+                                    <i class="material-icons">delete</i>
+                            </a>
                     </div>
+                </div>
                 <!-- Add Custom Domain section -->
-                    <div class="divider"></div>
-                    <div class="section">
-                        <div class="row">
-                            <div class="col s5 valign-wrapper">
-                                <h6 class="valign-wrapper"> Add Custom Domain:</h6>
-                            </div>
-                            <div class="col s5">
-                                <input id="CustomDomainInput" type="text" name="url" placeholder="Custom Domain URL">
-                            </div>
-                            <div class="col s2">
-                                <div class="right-align">
-                                    <a id="CustomDomainInputButton" class="btn-floating btn-small btn-jti tooltipped" data-position="left" data-delay="50" data-tooltip="Add Domain">
-                                        <i class="material-icons">add</i>
-                                    </a>
-                                </div>
-                            </div>
+                <div class="divider"></div>
+                <div class="section">
+                    <div class="row">
+                        <div class="col s5 valign-wrapper">
+                            <h6> Add Custom Domain:</h6>
+                        </div>
+                        <div class="col s5">
+                            <input id="customDomainInput" type="text" name="url" placeholder="Custom Domain URL">
+                        </div>
+                        <div class="col s2 right-align">
+                            <a id="customDomainInputButton" class="btn-floating btn-jti tooltipped" data-position="left" data-delay="50" data-tooltip="Add Domain">
+                                <i class="material-icons">add</i>
+                            </a>
                         </div>
                     </div>
-                <br>
-                <div class="divider"></div>
-                <br>
+                </div>
+                <div class="divider spaced"></div>
                 <!-- List of custom domains -->
-                <ul class="collection" align="left" id="customDomainsList">
+                <ul class="collection" id="customDomainsList">
                 </ul>
             </div>
         </div>
 
         <!-- Need this to be able to grab the sandbox window to compile the edit templates -->
         <iframe id="sandbox_window" src="sandbox.html" hidden></iframe>
+        <iframe id="sandbox_window2" src="sandbox.html" hidden></iframe>
```

Comment:
Is a second iframe necessary if they point to the same sandbox file?

Reply:
I don't think so